bot/discord.go

```diff
@@ -199,3 +200,21 @@ func dOutgoing(nick, channel, message string) {
 
 	dSession.ChannelMessageSend(chanID, fmt.Sprintf("**<%s>** %s", nick, message))
 }
+
+func getDisplayNameForMember(member *discord.Member) string {
+	if member.Nick != "" {
```

Comment:
Perhaps change the config struct to:
```go
type DiscordConfig struct {
 	Token        string "token"
    UseNicknames bool   "use_nicknames"
}
```

and change this condition to `conf.UseNicknames && member.Nick != ""`?

I don't plan to enable this for the bridge I'm running, but I'm happy to merge this if it's a config option.